src/routes/api/api-router.ts

```diff
@@ -91,6 +91,13 @@ ApiRouter.delete(
 	returnOnValidationError,
 	ApiInstallationDelete
 );
+ApiRouter.delete(
+	"/deleteInstallation/:installationId/:jiraHost/github-app-id/:gitHubAppId",
```

Comment:
/nit naming - do we need to call it `deleteInstallation` when the method is delete? so have `/installation`??
🤷

Reply:
If you expand the lines above a bit, you will see actually this is a duplicate route of an exisiting route (so that we can have the gitHubAppId)

Alternatively, I can modify the existing route, so that it reads
`"/deleteInstallation/:installationId/:jiraHost/(:gitHubAppId)?"`

But that routes is like `/:xxx/:xxx/:xxx`, doesn't looks very nice to me.

Reply:
And I think the route the bottom `ApiRouter.use("/:installationId", param("installationId").isInt(), returnOnValidationError, ApiInstallationRouter);` is more "prefered".

Although I seriously don't like it, as the route is like `DELETE /api/<any_int>/<any_string>`, which can easily match to any other route if not careful.

Reply:
Sorry @joshkay10, actually I am closing this one, as the existing route at the bottom of the file is the one supposed to be called. I'll fix pollinator anyway.

Reply:
yea good call better to leave as is at this point!!